src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    get info() {
+      return `name: ${this.name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
```

Comment:
Better change order, because initial values(like coords) better write before method. 
```suggestion
   coords: {
      x: 0,
      y: 0,
    },
    
    get info() {
      return `name: ${this.name}, chip version: ${version}, wheels: ${wheels}`;
    },
```